crates/bevy_pbr/src/light.rs

```diff
@@ -52,6 +53,84 @@ impl PointLightUniform {
     }
 }
 
+/// A Directional light.
+///
+/// Directional lights don't exist in reality but they are a good
+/// approximation for light sources VERY far away, like the sun or
+/// the moon.
+#[derive(Debug, Reflect)]
```

Comment:
should also derive `Clone`, `Copy`

Reply:
PointLight doesn't have those. Should I add them to it too while I'm at it?

Reply:
Yep!
Generally if something **can** be `Clone`/`Copy` they should be.